test/networkpolicy/open_policy_aws.yaml

```diff
@@ -50,7 +50,7 @@ data:
       - # rule: allow access to K8s apiserver (cluster-internal address)
         to:
         - ipBlock:
-            cidr: 172.16.0.0/28 # GKE master address range
+            cidr: 172.16.0.0/28 # GKE source address range
```

Comment:
above you called it "leading"?

Reply:
I guess both (yours and Andreas') comments are correct but I'll take Andreas' suggestion as it makes more sense in this context: https://github.com/SAP/stewardci-core/pull/176#discussion_r515048430